### ISSUE TITLE
filters.flow.worker_millis_per_event.lifetime marshal error in 1.8

### ISSUE THREAD

**Issue by VCCPlindsten** [kuskoman/logstash-exporter#372]
### Description of the Issue

There is an error logged by logstash-exporter 1.8.0 relating to float marshaling. See logs below.

Only basic info gets exported to prometheus endpoint.

### Version of logstash-exporter, or logstash-exporter Image

1.8.0

### Operating System/Environment

Logstash 8.15.0

### Logs

```sh
time=2024-10-10T12:49:14.807Z level=ERROR msg="executor failed" name=nodestats duration=25.028828ms err="json: cannot unmarshal string into Go struct field .pipelines.plugins.filters.flow.worker_millis_per_event.lifetime of type float64"
```

**Comment by fbs** [comment]
same here, seems like the output format is:

```
$ curl -s 'localhost:9600/_node/stats?pretty=true' | jq '.pipelines.main.plugins.filters[1].flow '
  "worker_millis_per_event": {
    "last_1_minute": "Infinity",
    "last_5_minutes": "Infinity",
    "last_15_minutes": "Infinity",
    "last_1_hour": "Infinity",
    "lifetime": "Infinity"
  }
```

while the type accepts only floats:

```
			Flow struct {
				WorkerUtilization struct {
					Current  float64 `json:"current"`
					Lifetime float64 `json:"lifetime"`
				} `json:"worker_utilization"`
				WorkerMillisPerEvent struct {
					Lifetime float64 `json:"lifetime"`
				} `json:"worker_millis_per_event"`
			} `json:"flow"`
```

**Comment by fbs** [comment]
Fixed in v1. Can you give it a go @VCCPlindsten ?

**Comment by VCCPlindsten** [comment]
Seems to work as intended. (Or I didn't look closely enough) Not getting the error from the same logstash instance with current v1 code.

**Comment by VCCPlindsten** [comment]
Fixed in 1.8.2

**Comment by satk0** [comment]
Reopen as it is still needed to be implemented in v2. I'll take care of that
